win32ss/user/ntuser/winpos.c

```diff
@@ -2715,9 +2715,15 @@ co_WinPosShowWindow(PWND Wnd, INT Cmd)
          Parent = Wnd->spwndParent;
          if (UserIsDesktopWindow(Parent))
              Parent = NULL;
-         if (Parent) UserRefObjectCo(Parent, &Ref);
+         if (Parent)
+         {
+            UserRefObjectCo(Parent, &Ref);
```

Comment:
Indentation looks to miss 1 space...
If you add braces here, then as well add them to previous if() too.

Reply:
OK.